third_party/OpenAPI/api/version.swagger.yaml

```
@@ -0,0 +1,274 @@
+swagger: "2.0"
+info:
+  title: api/version.proto
+  version: "1.0"
+schemes:
+  - http
+  - https
+consumes:
+  - application/json
+produces:
+  - application/json
+paths:
+  /versions/v1/{product}/{operatorVersion}/{apply}:
+    get:
+      summary: Specific version
+      description: Return specific product version
+      operationId: VersionService_Apply
+      responses:
+        "200":
+          description: A successful response.
+          schema:
+            $ref: '#/definitions/versionVersionResponse'
+        default:
+          description: An unexpected error response
+          schema:
+            $ref: '#/definitions/googlerpcStatus'
+      parameters:
+        - name: product
+          in: path
+          required: true
+          type: string
+        - name: operatorVersion
+          in: path
+          required: true
+          type: string
+        - name: apply
+          in: path
+          required: true
+          type: string
+        - name: databaseVersion
+          in: query
+          required: false
+          type: string
+        - name: kubeVersion
+          in: query
+          required: false
+          type: string
+        - name: platform
+          in: query
+          required: false
+          type: string
+        - name: pmmVersion
+          in: query
+          required: false
+          type: string
+        - name: backupVersion
+          in: query
+          required: false
+          type: string
+        - name: namespaceOid
+          in: query
+          required: false
+          type: string
+        - name: customResourceOid
```

can we rename a bit?
```suggestion
        - name: namespaceUID
          in: query
          required: false
          type: string
        - name: customResourceUID
```

reason: field name is "uid"
```
% kubectl get pxc cluster1 -o yaml | grep uid
  uid: 7fe4023e-c078-11ea-9608-42010a9c0052

% kubectl get ns default -o yaml | grep uid
  uid: 6448094f-af9e-11ea-a12d-42010a9c00ee
```

---

https://github.com/Percona-Lab/percona-version-service/pull/1/commits/4e735088c442a1e6cdd20bcd3fbeea896aa1cf63